spec/datetimeformat.html

```diff
@@ -113,12 +114,16 @@
           1. Let _p_ be Get(_bestFormat_, _prop_).
           1. If _p_ not *undefined*, then
             1. Set _dateTimeFormat_'s internal slot whose name is the Internal Slot column of the row to _p_.
-        1. Let _hr12_ be ? GetOption(_options_, *"hour12"*, *"boolean"*, *undefined*, *undefined*).
         1. If _dateTimeFormat_ has an internal slot [[Hour]], then
-          1. If _hr12_ is *undefined*, then
-            1. Let _hr12_ be Get(_dataLocaleData_, *"hour12"*).
-          1. Set _dateTimeFormat_.[[Hour12]] to _hr12_.
+          1. Let _hc_ be ? GetOption(_options_, *"hourCycle"*, *"string"*, &laquo; *"h12"*, *"h24"* &raquo;, *undefined*).
```

Comment:
> What's the reason to wait? 

Supporting more options requires algorithm changes around hour token formatting.

Ok, I'll work on that, but this will increase the size of this patch substantially and require a re-review.